src/main/docs/guide/server.adoc

```diff
@@ -108,6 +108,30 @@ and then the order will be reversed when sending the response message.
 Request -> 1 -> 2 -> 3 -> business logic -> 3 -> 2 -> 1 -> Response
 ----
 
+=== Health checks
+If the management dependency ("io.micronaut:micronaut-management") is added, then Micronaut’s Health Endpoint can be used to expose the health status of the Grpc server.
+
+For example if grpc is running the /health endpoint will return:
```

Comment:
Could you change the references to gRPC to be consistently cased. I believe the correct casing is `gRPC`.

Reply:
Fixed

Reply:
Done here. I'll create a new PR to change it in all the docs.